fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/ReindexResourceDAO.java

```diff
@@ -283,12 +283,10 @@ public void updateParameters(String tablePrefix, List<ExtractedParameterValue> p
 
         // existing resource, so need to delete all its parameters
         // delete composites first, or else the foreign keys there restrict deletes on referenced tables
-        deleteFromParameterTable(connection, tablePrefix + "_composites", logicalResourceId);
         deleteFromParameterTable(connection, tablePrefix + "_str_values", logicalResourceId);
         deleteFromParameterTable(connection, tablePrefix + "_number_values", logicalResourceId);
         deleteFromParameterTable(connection, tablePrefix + "_date_values", logicalResourceId);
         deleteFromParameterTable(connection, tablePrefix + "_latlng_values", logicalResourceId);
-        deleteFromParameterTable(connection, tablePrefix + "_token_values", logicalResourceId);
```

Comment:
cool - I didn't realize we. were getting. rid of. this as. well.

Reply:
Yep, getting rid of this and composites will save us quite a bit of space too.